### PR TITLE
Move includeText to QueryFunctions

### PR DESCRIPTION
**Pull request by lbschanno**
The filter:includeText function is not truly evaluation-only, and as
such, does not truly belong in EvaluationPhaseFilterFunctions. Move it
to QueryFunctions.

Done per a request from @ivakegg

### CODE REVIEW

File: warehouse/query-core/src/main/java/datawave/query/jexl/functions/QueryFunctionsDescriptor.java [hunk 1]
```diff
@@ -43,24 +48,44 @@ public QueryJexlArgumentDescriptor(ASTFunctionNode node, String namespace, Strin
...
+                    return TRUE_NODE;
+            }
+        }
+        
+        private JexlNode getTextIndexQuery() {
+            JexlNode node0 = args.get(0);
+            final String value = args.get(1).image;
+            if (node0 instanceof ASTIdentifier) {
+                final String field = JexlASTHelper.deconstructIdentifier(node0.image);
+                return JexlNodeFactory.buildNode((ASTERNode) null, field, value);
```
Comment (ivakegg) [inline warehouse/query-core/src/main/java/datawave/query/jexl/functions/QueryFunctionsDescriptor.java]: This should be an EQ node

File: warehouse/query-core/src/main/java/datawave/query/jexl/functions/QueryFunctionsDescriptor.java [hunk 2]
```diff
@@ -43,24 +48,44 @@ public QueryJexlArgumentDescriptor(ASTFunctionNode node, String namespace, Strin
...
+            } else {
+                // node0 is an Or node or an And node
+                // copy it
+                JexlNode newParent = JexlNodeFactory.shallowCopy(node0);
+                int i = 0;
+                for (ASTIdentifier identifier : JexlASTHelper.getIdentifiers(node0)) {
+                    String field = JexlASTHelper.deconstructIdentifier(identifier.image);
+                    JexlNode kid = JexlNodeFactory.buildNode((ASTERNode) null, field, value);
```
Comment (ivakegg) [inline warehouse/query-core/src/main/java/datawave/query/jexl/functions/QueryFunctionsDescriptor.java]: This should be an EQ node

**Review by ivakegg — CHANGES_REQUESTED** [review]
In addition to my comments, you are missing a change in the datawave.query.language.functions.jexl.Text class.  toString needs to output f:includeText instead of filter:includeText

File: warehouse/query-core/src/main/java/datawave/query/jexl/functions/QueryFunctionsDescriptor.java (hunk 1 above)
Comment (lbschanno) [reply warehouse/query-core/src/main/java/datawave/query/jexl/functions/QueryFunctionsDescriptor.java]: Fixed.

File: warehouse/query-core/src/main/java/datawave/query/jexl/functions/QueryFunctionsDescriptor.java (hunk 2 above)
Comment (lbschanno) [reply warehouse/query-core/src/main/java/datawave/query/jexl/functions/QueryFunctionsDescriptor.java]: Fixed.

**Review by ivakegg — DISMISSED** [review]

**Review by apmoriarty — APPROVED** [review]

**Review by apmoriarty — DISMISSED** [review]

**Review by apmoriarty — DISMISSED** [review]

**Review by ivakegg — DISMISSED** [review]

**Comment by ivakegg** [comment]
I merely resolved conflicts with a rebase and the force pushed.  There were already two approvals for this PR.

**Review by apmoriarty — APPROVED** [review]

**Pull request merged**